server.cfg

```diff
@@ -39,9 +39,9 @@ ensure hardcap
 ensure baseevents
 
 # QBCore & Extra stuff
-ensure qb-core
 ensure ox_lib
 ensure ox_target
+ensure qb-core
```

Comment:
Doesn't ox_target need to be after qb-core? I think the qb compatibility files rely on it

Reply:
You are correct, forgot to check this on client.